### PR TITLE
Fetch dapp staking user transactions history from SubsSquid Giant Squid

### PR DESCRIPTION
**Pull request by bobo-k2**
Currently we have indexer which indexes dapp staking user transactions history for Astar, but we don't have it for Shiden and Shibuya. Since process of testing/debugging/indexing is a quite time consuming I decided to use SubsSquid data to fetch the history for all our networks. @gluneau thanks for the idea, you did the same for gas station.

This PR contains logic to fetch and transform SubSquid transactions data to data format already used by Token API

### CODE REVIEW

File: src/containertypes.ts [hunk 1]
```diff
@@ -13,4 +13,6 @@ export const ContainerTypes = {
     MonthlyActiveWalletsService: 'MonthlyActiveWalletsService',
     DappsStakingStatsService: 'DappsStakingStatsService',
     DappRadarService: 'DappRadarService',
+    GiantSquidService: 'GiantSquidService',
+    A: 'A',
```
Comment (gluneau) [inline src/containertypes.ts]: I don't think this is needed.  or it needs a glorious comment!

**Review by gluneau — CHANGES_REQUESTED** [review]
Are there some tests for this? Or is it not needed?

Is this a good endpoint to test the veracity of the response? 
`curl 'http://127.0.0.1:5001/astar-token-api/us-central1/app/api/v1/astar/dapps-staking/stats/user/{userAddress}/90%20days`

**Comment by impelcrypto** [comment]
It seems that stats history for Shiden is not working. I staked 10 mins ago, but the API hasn't updated for the transaction yet. 

https://astar-token-api--pr102-feat-giant-squid-afm79aro.web.app/api/v1/shiden/dapps-staking/stats/user/YyaqqvoACtudUvrqBXp7RxyAHkccvtj5BuESchUWY8EvBMF/1%20years

https://shiden.subscan.io/extrinsic/4753455-2

image

File: src/containertypes.ts (hunk 1 above)
Comment (gluneau) [reply src/containertypes.ts]: I’m talking about Line 17

**Review by impelcrypto — APPROVED** [review]

File: src/containertypes.ts (hunk 1 above)
Comment (bobo-k2) [reply src/containertypes.ts]: Thanks fixed, also added a test

**Review by gluneau — APPROVED** [review]
Lgtm

**Pull request merged**